### PR TITLE
--append_numbers flag to append numbers to part names in library

### PR DESCRIPTION
**Pull request by Kamoool**
Hello,
I have added --append_numbers flag to append part numbers to names in the library. That helps a little bit when searching in the library, because of working on part numbers. I have changed the LCSC number property to be hidden by default because IMHO it's not really needed straight away on the schematic. Also, I have added a Name property that will keep the name of the component without the LCSC part number.

### CODE REVIEW

**Comment by TousstNicolas** [comment]
Hi,

**Comment by TousstNicolas** [comment]
Is there a reason you use the property "Name" over  "Value" to  display ? It is a detail, but the first thing I noticed is that KiCad display the "Name" field in a different color than "Value" (which is what all other components use). 
After that, a small rebase to clean and it should be good.

File: JLC2KiCadLib/schematic/schematic.py [hunk 1]
```diff
@@ -53,8 +55,12 @@ class kicad_schematic:
             .replace(" ", "_")
             .replace(".", "_")
         )
+        component_description = data["result"]["description"]
```
Comment (TousstNicolas) [inline JLC2KiCadLib/schematic/schematic.py:58]: unused variable,  I think it is coming from your other  PR. You can remove it in this one.

**Comment by Kamoool** [comment]
Value in Kicad is used in general as a name in the library to search for components. When the new flag is used it is appended with C...... number, then using the filter in the schematic library you can find the part with either C.... number and part name as well. I have added the Name property that is hidden when --append_numbers flag is not used and shown (Value is hidden) when the flag is used to show only part name, without C number, on the schematic.

File: JLC2KiCadLib/schematic/schematic.py (hunk 1 above)
Comment (Kamoool) [reply JLC2KiCadLib/schematic/schematic.py:58]: Exactly, missed that.

**Comment by TousstNicolas** [comment]
If it is only used to be able to search it using the Cxx number,  it is easier to use the "keywords" property, which is here for this exact purpose.

**Comment by TousstNicolas** [comment]
yes I agree, it can be done by default. I will add handle it, will be easier than opening a new PR for that small change.

**Pull request closed without merging**